src/components/Comments/ListComment.js

```diff
@@ -23,6 +24,7 @@ const ListComment = ({
         <View key={`p${index}`}>
           {item.user ? (
             <Comment
+              feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `feedId` as a prop to the `Comment` component is a good change, but make sure that the `Comment` component is updated to handle this new prop and that all instances where the `Comment` component is used are updated to pass this new prop.
```diff
+  feedId={feedId}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->